SSOT/DeleteCSTData.sql

```diff
@@ -0,0 +1,5 @@
+----Drop the foreign keys and primary keys on Id column in NOC table.

```

Comment:
Please rename this file. `DeleteCSTData` is scary and will raise unnecessary questions.